### ISSUE TITLE
Virtual keyboard accessibility

### ISSUE THREAD

**Issue by maxshuty** [hodgef/simple-keyboard#907]
**Is your feature request related to a problem?**
Keyboard only users cannot navigate into and activate virtual keyboard keys

**Describe the solution you'd like**
Keyboard only users can tab into and activate the virtual keyboard keys. Additionally the proper aria labels (if needed) would be applied to the `button`s so that screen readers can read them out for visually impaired users.

On the surface it seems silly: "why would a keyboard only user need to access a virtual keyboard?"
However here is an example of how this can become problematic:
Let's say we have a website that is an online digital library. This library may contain media from any number of languages. A user may have an English keyboard and they want to search for a book in Chinese. The user then opens the Chinese simple-keyboard and types in their Chinese characters, then they submit their search and get their results back.

If this same user is a keyboard-only user they will not be able to search for their book in Chinese.

I cannot think of a valid reason why a keyboard only user may want to user a virtual keyboard in their same language, but there may be some use cases I haven't considered as I _am not_ a person impacted by this.

The solution? Make the simple-keyboard accessible by adding the proper `aria` attributes and `tabindex` to the `hg-button`s. To me it would seem reasonable that the `enter` key would do the same action that clicking a key currently does. However, this could be problematic if someone has a website with a form that is submitted by the enter key. I'm open to hearing thoughts on this and other ideas. Perhaps arrow keys and spacebar could be another alternative? (Kind of like how checkboxes are checked and how radio buttons are toggled between.)

Depending on how simple-keyboard is built, this whole request may be too tough to overcome if the underlying code is built around everything being shaped a certain way, but this feature is something my use case would require and it would be really nice to not have to get hacky with it.

**Describe alternatives you've considered**
I'm not totally sure there are any alternatives to this problem short of getting really hacky and adding a `tabindex` to the `hg-button`s and event listeners for keydown.

**Additional context**
Please refer to [WebAIM Keyboard Accessibility](https://webaim.org/techniques/keyboard/) for more information on keyboard accessibility.

**Comment by hodgef** [comment]
Hey @maxshuty,

My general approach with simple-keyboard is to provide flexibility so developers can then adapt the keyboard to their use-cases. This really helps to keep maintenance at manageable levels.

On the topic of accessibility, I have added the ability to add attributes to buttons as requested here: https://github.com/hodgef/simple-keyboard/issues/196

I think if you use [`buttonAttributes`](https://hodgef.com/simple-keyboard/documentation/options/buttonattributes/) to add the desired attributes and use options like [`useButtonTag`](https://hodgef.com/simple-keyboard/documentation/options/usebuttontag/), etc. you can reach satisfactory results. Let me know if you're coming across any difficulties with that approach.

Regards,
Francisco Hodge
